assets/scripts/whichx.js

```diff
@@ -158,18 +201,29 @@ function WhichX(config) {
         return typeChance * (type.tcount / total.tcount);
     }
 
-    // A non-zero prior estimate to prevent 0 based probability calculations.
+    /**
+     * A non-zero prior estimate to prevent 0 based probability calculations.
+     * @returns {number} The non-zero probablity
+     */
     function mEstimate() {
         var total = typesMap.total;
         return 1 / (total.wordTotal * 100);
     }
 
-    // Process the description into an array of pertinent standardized lower case words.
+    /**
+     * Process the description into an array of pertinent standardized lower case words.
+     * @param {string} description The description to analyse
+     * @returns {string[]} The list of normalized words contained in the description
+     */
     function processToArray(description) {
         var i = 0;
         if (typeof description === "string") {
             // Remove special characters.
-            description = description.replace(/[^a-zA-Z ]/g, "");
+            description = description
+                .normalize("NFD")
```

Comment:
`.normalize()` is not available in Internet Explorer. Before this addition WhichX supports IE9 and above. Can you find an alternative? I appreciate this may be a big ask.

Reply:
The current solution is to rely on [unorm](https://github.com/walling/unorm/). Is it ok? Other solution would require to maintain an exhaustive list of letter characters from all languages... We could also decide to keep all special characters in the string except for punctuation and maintain a list of punctuation characters... Tell me what you prefer

Reply:
One of the original goals of this project was to leave it without any dependencies (at least not in the deliverable). Additionally, unorm looks like it needs a shim to work pre-ECMA 5. I'm not willing to start using dependencies in the deliverable, how-ever we could simply check if the function is defined before calling it.

```
if (typeof(description.normalize) === typeof(Function)) {
     description = description.normalize("NFD");
}
```